adapters/paj.paj

```diff
@@ -0,0 +1,2 @@
+$var = "Connection established";
+print($var);
```

Comment:
There is no newline at the end of this file. ian-db follows rigurous quality standards. Please correct.

Reply:
![image](https://user-images.githubusercontent.com/9765622/94745734-8c060e80-037b-11eb-8b57-71b434d4c3a1.png)

Reply:
rocket emoji